raytracing/drawing.py

```diff
@@ -0,0 +1,324 @@
+import matplotlib.pyplot as plt
+from matplotlib import patches, transforms
+from matplotlib import text as mplText
+import matplotlib.path as mpath
+from typing import List
+import numpy as np
+import math
+
+
+class Drawing:
```

Comment:
I think the class should be called `Graphic` or `Graphics`

Reply:
Graphic yes, it makes sense